810-well-known-custom-extensions.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Well known custom extensions
+weight: 810
+---
+
+# Well known custom actions
+
+This section is non-normative, but is here to propose a common set of custom extensions a bundle MAY define, and that CNAB tools MAY implement.
+
+A bundle indicates that it uses a custom extension by including them in its custom section (as defined in [Custom Extensions](101-bundle-json.md#custom-extensions)).
+
+## Dependencies
+
+See the [Dependencies Specification](500-CNAB-dependencies.md).
+
+## Parameter Sources
+
+A custom bundle extension, `io.cnab.parameterSources`, MAY be defined that provides additional guidance for how a runtime MAY determine the default value of a [parameter][parameter]. The parameter source extension does not cover how the runtime should set the value of a parameter because each implemenation may have different information to take into account when determining the final value of a parameter.
```

Comment:
Nit: in [well-known-custom-actions](https://github.com/cnabio/cnab-spec/blob/b5ed2f30bdc51fb9953fcd74d6b13d83c4165849/804-well-known-custom-actions.md#well-known-custom-actions
) we appear to be using dasherized names, eg. `io.cnab.dry-run`.
```suggestion
A custom bundle extension, `io.cnab.parameter-sources`, MAY be defined that provides additional guidance for how a runtime MAY determine the default value of a [parameter][parameter]. The parameter source extension does not cover how the runtime should set the value of a parameter because each implementation may have different information to take into account when determining the final value of a parameter.
```